src/components/Loader/Loader.tsx

```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+
+import classNames from 'classnames';
+
+import styles from './Loader.module.css';
+
+export type Size = 'sm' | 'md' | 'lg' | 'xl';
+
+interface LoaderProps extends React.HTMLAttributes<HTMLDivElement> {
+  size?: Size;
+}
+
+const Loader: React.FC<LoaderProps> = ({ size = 'lg', ...rest }) => {
+  const mainClass = classNames(styles.loader);
+  const iconClass = classNames(
+    styles['loader-icon'],
+    styles[`size--${size}`],
+    rest.className
+  );
+
+  const iconProp = {
+    ...rest,
```

Comment:
Need a second opinion on spreading custom props in the nested div rather than the wrapper div.
Attaching it to the nested div allows consumer to pass in custom sizes and have it override the default `sm`, `md`, `lg`, `xl` options.
An alternative option would be to pass a numeric to the size prop rather than enum strings. Then the custom prop can be attached to the wrapper div. This will break away from the enum string pattern used in trendkite-ui

Reply:
I don't think I'd call this an "icon", I might just say "Loader" or something, since our icons are usually svgs that you can change colors on.

Reply:
Updated to loader to render with one div and custom props would get spread on that div.
I kept the css that renders the loader's location in the center of its parent html element. 
That way consumers can adjust the loader's horizontal positioning with a parent html element that has a defined width and margin.
They could also override the default horizontally centered loader position via style prop if consumers do not have control over the parent html element.